packages/mainmenu-extension/src/index.ts

```diff
@@ -485,6 +493,17 @@ function createTabsMenu(app: JupyterLab, menu: TabsMenu): void {
     return { command: commandID };
   };
 
+  let previousId = '';
+
+  // Command to toggle between the current
+  // tab and the previously used tab.
+  if (!commands.hasCommand(CommandIDs.activatePreviouslyUsedTab)) {
```

Comment:
This `if` statement check is not necessary, and I think it can be safely removed, since the command will only be added once on plugin activation.